### PR TITLE
fix #610 privacy CA now known as attestation CA

### LINKED ISSUE #610: Per TCG: "privacy CA" is now "Attestation CA"
Note that if one searches the TCG's  TPM 2.0 specs for "privacy CA", one will not find it. 

I went looking and found this in https://trustedcomputinggroup.org/wp-content/uploads/IWG_CMC_Profile_Cert_Enrollment_v1_r7.pdf

>2.2 Attestation CA
The primary Certification Authority (CA) associated with the TPM has been referred to in previous TCG publications as either the “Privacy CA” (PCA) or “Platform CA”. In an effort to separate the issues of privacy from those of attestation, that designation is replaced here with the term “Attestation CA” (ACA). The role and operation of the ACA is described in detail in section 3, below.

Indeed "attestation CA" is used in e.g. in [TPM 2.0 Part 1 Architecture](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-1-Architecture-01.38.pdf).

### PR DESCRIPTION
**Pull request by equalsJeffH**
fixes #610

[ in case anyone noticed (unlikely, I realize): the branch name this PR is based on is incorrect, it should be `jeffh-fix-610-privacy-ca-cites`, but oh well it is not. too much trouble to change it at this point...]

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/762.html" title="Last updated on Jan 30, 2018, 11:45 PM GMT (b380a03)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/762/b0cae5e...b380a03.html" title="Last updated on Jan 30, 2018, 11:45 PM GMT (b380a03)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -2645,12 +2646,14 @@ WebAuthn supports multiple attestation types:
...
-:: In this case, the Authenticator owns an authenticator-specific (endorsement) key. This key is used to securely communicate
-    with a trusted third party, the Privacy CA.  The Authenticator can generate multiple attestation key pairs and asks the
-    Privacy CA to issue an attestation certificate for it. Using this approach, the Authenticator can limit the exposure of the
-    endorsement key (which is a global correlation handle) to Privacy CA(s). Attestation keys can be requested for each [=public
-    key credential=] individually.
+: <dfn>Attestation CA</dfn>
+:: In this case, an [=authenticator=] is based on a Trusted Platform Module (TPM) and holds an authenticator-specific
+    "endorsement key" (EK). This key is used to securely communicate with a trusted third party, the [=Attestation CA=]
+    [[!TCG-CMCProfile-AIKCertEnroll]] (formerly known as a "Privacy CA"). The [=authenticator=] can generate multiple
+    attestation identity key pairs (AIK) and requests an [=Attestation CA=] to issue an attestation identity certificate (AIK)
```
Comment (emlun) [inline index.bs]: Should the latter AIK here be AIC?

File: index.bs [hunk 2]
```diff
@@ -2645,12 +2646,14 @@ WebAuthn supports multiple attestation types:
...
+    for each. Using this approach, such an [=authenticator=] can limit the exposure of the EK (which is a global correlation
+    handle) to Attestation CA(s). AIKs can be requested for each [=authenticator=]-generated [=public key credential=]
```
Comment (emlun) [inline index.bs]: Ditto

File: index.bs [hunk 3]
```diff
@@ -1746,14 +1746,15 @@ during credential generation.
     *   <dfn>none</dfn> - indicates that the [=[RP]=] is not interested in [=authenticator=] [=attestation=].
         The client MAY replace the [=AAGUID=] and [=attestation statement=] generated
         by the authenticator with meaningless client-generated values. For example, in order to avoid having to obtain
-        [=user consent=] to relay uniquely identifying information to the [=[RP]=], or to save a roundtrip to a Privacy CA.
+        [=user consent=] to relay uniquely identifying information to the [=[RP]=], or to save a roundtrip to a
```
Comment (selfissued) [inline index.bs]: Change "a" to "an"

**Comment by jcjones** [comment]
It looks like the CI build didn't run... is Anonymization CA defined anywhere? I don't see it.

**Comment by equalsJeffH** [comment]
@jcjones 
> is Anonymization CA defined anywhere? I don't see it.

see https://github.com/w3c/webauthn/pull/762/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR4448

> It looks like the CI build didn't run...

it appears it was that "used only once" `allowCredentialDescriptorList` warning that we fixed earlier today. now fixed in master and this branch.  looks like travisCI is now running....

thx for headzup :)

File: index.bs (hunk 1 above)
Comment (equalsJeffH) [reply index.bs]: "AIC" is not used in [[!TCG-CMCProfile-AIKCertEnroll]], so I'm thinking not....

File: index.bs (hunk 2 above)
Comment (equalsJeffH) [reply index.bs]: Ditto ;-)

File: index.bs (hunk 3 above)
Comment (equalsJeffH) [reply index.bs]: fixed, see https://github.com/w3c/webauthn/pull/762/commits/0cbccd00839e874b81409414256c6071e322f6aa

**Comment by equalsJeffH** [comment]
bummer.  

> FATAL ERROR: Couldn't find 'TCG-CMCProfile-AIKCertEnroll' in bibliography data. 

I don't know what's going.....wait...

https://www.specref.org/?q=TCG-CMCProfile-AIKCertEnroll

....shows that the reference is in specref.org...so i think this error is a hint that we need to update our cached repo-side build stuff such that we have the latest specref data ?

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs]: Ok, I just wanted to thought it looked strange that the same acronym was used for two different things.

**Comment by jcjones** [comment]
@equalsJeffH: I've updated the spec-data, but to tell in the PR we'd need to rebase the PR, which I can't do from this interface.

**Comment by equalsJeffH** [comment]
@jcjones: I've updated the spec-data...
thanks!  I merged-from-master https://github.com/w3c/webauthn/pull/762/commits/cb6ba298190894cbfc01ba6779921f2d5860d36b ...waiting to see if the travis build is successful...

**Comment by equalsJeffH** [comment]
it would be nice to get an OK from Chrome folk since I invented the "anonymization CA" name for the entity they have so far been refering to as a "Privacy CA" (which would end up being confusing since that name was originally used by TCG specs for an entity that behaves somewhat differently and now is itself renamed to "Attestation CA")

**Comment by agl** [comment]
If we end up using a privacy CA, we're probably going to call it a privacy CA irrespective of whatever is in the spec. I think the term is pretty well established for "thing that checks a specific certificate and issues a general one saying that it checked it". I don't see that we need another term for the same thing.

File: index.bs (hunk 1 above)
Comment (equalsJeffH) [reply index.bs]: thx, you did identify a problem -- should be using term "AIK certificate" here.

File: index.bs (hunk 2 above)
Comment (equalsJeffH) [reply index.bs]: I am thinking usage of AIK here is nominally correct.

**Comment by equalsJeffH** [comment]
@agl wrote:
> If we end up using a privacy CA, we're probably going to call it a privacy CA irrespective of whatever is in the spec.

Thx, understood. I prefer to keep these terms separate in the spec, and y'all can call such functionality whatever you want, if it is materialized.

**Comment by selfissued** [comment]
Agreed.  I think this is ready to merge.

File: index.bs (hunk 2 above)
Comment (emlun) [reply index.bs]: Yeah, this looks better!

**Pull request merged**